Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: tests
```

Comment:
Please remove build changes and put them in a separate PR for review and discussion.